### PR TITLE
fix(login): centered login on larger viewports

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/2729

### CODE REVIEW

File: src/patternfly/components/Login/login.scss [hunk 1]
```diff
@@ -117,7 +116,6 @@
 }
 
 .pf-c-login__container {
-  width: 100%;
```
Comment (rhamilto) [inline src/patternfly/components/Login/login.scss:120]: This change resulted in a regression where the container can be smaller than the viewport at mobile.

oauth-openshift apps rhamilto devcluster openshift com_oauth_authorize_client_id=console redirect_uri=https%3A%2F%2Fconsole-openshift-console apps rhamilto devcluster openshift com%2Fauth%2Fcallback response_type=code scope=user%3Afull  (1)

www patternfly org_v4_documentation_react_components_loginpage_basic(iPhone 6_7_8)


I'm pretty sure we're going to want this back.

Comment (mcoker) [reply src/patternfly/components/Login/login.scss:120]: Thanks for catching that Robb, I'll try and look at https://github.com/patternfly/patternfly/issues/2913 later today.

Comment (mcoker) [reply src/patternfly/components/Login/login.scss:120]: I've put up a PR and added a comment to the code so hopefully that shouldn't be removed again. https://github.com/patternfly/patternfly/pull/2917

**Pull request merged**